.github/workflows/docgen.yaml

```diff
@@ -0,0 +1,22 @@
+name: panvimdoc
+
+on: [push]
+
+jobs:
+  docs:
+    runs-on: ubuntu-latest
+    name: pandoc to vimdoc
+    steps:
+      - uses: actions/checkout@v2
+      - name: panvimdoc
+        uses: kdheepak/panvimdoc@main
+        with:
+          vimdoc: sidebar
+          description: ""
+          pandoc: "doc/README.md"
+          toc: true
+          version: "NVIM v0.6.0"
+      - uses: stefanzweifel/git-auto-commit-action@v4
+        with:
+          commit_message: "docs: auto generate docs"
+          branch: ${{ github.head_ref }}
```

Comment:
@GustavoKatel I took a test, see https://github.com/aspeddro/sidebar.nvim/pull/1.

I didn't quite understand the flow. When the PR is opened you can look at `docs/general.md` and see if everything is ok, and in case of error the CI breaks.

It seems that it is not possible to open a PR in upstream, only in the fork itself.

Reply:
@aspeddro I think this is perfect! we can test better when we merge

I'm going to wait for @davysson as he was the one working first in the vimdoc stuff